plugins/source/alicloud/codegen/models/bss.go

```diff
@@ -0,0 +1,64 @@
+package models
+
+type BillOverview struct {
```

Comment:
Mostly out of curiosity, what is the difference between these structs and the ones provided by the Alicloud SDK? Just wondering why (or whether) we need it